src/interactive/cli.ts

```diff
@@ -122,20 +123,20 @@ export async function interactiveCli(baseargv: string[]) {
       const argsExport = [...baseargv.slice(0, 2), "transaction", `export${taskConstants[task].slice(-2)}`, '-a', `${amount.amount}`, `--env-path=${walletProperties.path}`, `--network=${walletProperties.network}`, "--get-hacked"]
       // ask for fees if its exportCP transaction
       if (taskConstants[task].slice(0, 1) == 'C') {
-        const exportBaseFee = await getFeesBasedOnChainForPrivateKey(taskConstants[task].slice(0, 1), walletProperties.path, walletProperties.network)
-        const exportFees = await prompts.fees(exportBaseFee);
+        const exportFees = await prompts.fees(DEFAULT_EVM_TX_FEE);
         argsExport.push('-f', `${exportFees.fees}`)
+        await waitFinalize<any>(contextEnv(walletProperties.path, walletProperties.network), program.parseAsync(argsExport))
+        console.log(chalk.green("Transaction finalized!"))
+      } else {
+        await program.parseAsync(argsExport)
       }
-      console.log("Please approve export transaction")
-      await program.parseAsync(argsExport)
       const argsImport = [...baseargv.slice(0, 2), "transaction", `import${taskConstants[task].slice(-2)}`, `--env-path=${walletProperties.path}`, `--network=${walletProperties.network}`, "--get-hacked"]
       // ask for fees if its importTxPC
       if (taskConstants[task].slice(0, 1) == 'P') {
-        const importBaseFee = await getFeesBasedOnChainForPrivateKey(taskConstants[task].slice(1, 2), walletProperties.path, walletProperties.network)
-        const exportFees = await prompts.fees(importBaseFee);
+        const exportFees = await prompts.fees(DEFAULT_EVM_TX_FEE);
+        console.log("export fees", exportFees.fees)
```

Comment:
This should be removed or log with proper formatting

Reply:
We will add in our pr. this is sync from the main branch from flare